daft/dataframe/dataframe.py

```diff
@@ -1370,27 +1638,22 @@ def explode(self, *columns: ColumnInputType) -> "DataFrame":
         Exploding Null values or empty lists will create a single Null entry (see example below).
 
         Example:
-            >>> df = daft.from_pydict({
-            >>>     "x": [[1], [2, 3]],
-            >>>     "y": [["a"], ["b", "c"]],
-            >>>     "z": [1.0, 2.0],
-            >>> ]})
-            >>>
-            >>> df.explode(col("x"), col("y"))
-            >>>
-            >>> # +------+-----------+-----+      +------+------+-----+
-            >>> # | x    | y         | z   |      |  x   |  y   | z   |
-            >>> # +------+-----------+-----+      +------+------+-----+
-            >>> # |[1]   | ["a"]     | 1.0 |      |  1   | "a"  | 1.0 |
-            >>> # +------+-----------+-----+  ->  +------+------+-----+
-            >>> # |[2, 3]| ["b", "c"]| 2.0 |      |  2   | "b"  | 2.0 |
-            >>> # +------+-----------+-----+      +------+------+-----+
-            >>> # |[]    | []        | 3.0 |      |  3   | "c"  | 2.0 |
-            >>> # +------+-----------+-----+      +------+------+-----+
-            >>> # |None  | None      | 4.0 |      | None | None | 3.0 |
-            >>> # +------+-----------+-----+      +------+------+-----+
-            >>> #                                 | None | None | 4.0 |
-            >>> #                                 +------+------+-----+
+            >>> import daft
+            >>> df = daft.from_pydict({"x": [[1], [2, 3]],"y": [["a"], ["b", "c"]],"z": [[1.0],[2.0, 2.0],]})
```

Comment:
Can you open up the input dict into multiple lines? (Like before). It's just easier to see